pandas/compat/__init__.py

```diff
@@ -12,7 +12,6 @@
 import sys
 import warnings
 
-PY35 = sys.version_info[:2] == (3, 5)
 PY36 = sys.version_info >= (3, 6)
```

Comment:
can remove this entirely in a followup I think (by-definition PY36 will always be true)